<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Compresseur 4K</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btnSelectImage">
      <property name="text">
       <string>Sélectionner une image</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutCompression">
      <item>
       <widget class="QPushButton" name="btnSLIC">
        <property name="text">
         <string>Compresser avec SLIC</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSDGT">
        <property name="text">
         <string>Compresser avec SDGT</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutOutput">
      <item>
       <widget class="QPushButton" name="btnSelectOutput">
        <property name="text">
         <string>Définir le dossier de destination</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditOutputPath">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutImages">
      <item>
       <widget class="QGraphicsView" name="graphicsViewImageInput"/>
      </item>
      <item>
       <widget class="QGraphicsView" name="graphicsViewImageOutput"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutMetrics">
      <item>
       <widget class="QLabel" name="labelPSNRTitle">
        <property name="text">
         <string>PSNR :</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelPSNRValue">
        <property name="text">
         <string>-- dB</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelCompressionTitle">
        <property name="text">
         <string> Taux de compression :</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelCompressionValue">
        <property name="text">
         <string>--</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
